nbgrader/apps/extensionapp.py

```diff
@@ -1,299 +1,35 @@
-        extra_args = self.extra_args[:]
-
-        # install the create_assignment extension
-        if len(extra_args) == 0 or "create_assignment" in extra_args:
-            self.log.info("Installing create_assignment extension")
-            self.extra_args = [os.path.join(nbextensions_dir, 'static', 'create_assignment')]
-            self.install_extensions()
-
-        # install the assignment_list extension
-        if sys.platform != 'win32' and (len(extra_args) == 0 or "assignment_list" in extra_args):
-            self.log.info("Installing assignment_list extension")
-            self.extra_args = [os.path.join(nbextensions_dir, 'static', 'assignment_list')]
-            self.install_extensions()
-
-
-class ExtensionUninstallApp(UninstallNBExtensionApp, NbGrader):
-
-    name = u'nbgrader-extension-uninstall'
-    description = u'Uninstall the nbgrader extensions'
-
-    examples = """
-
-    To uninstall all the nbgrader extensions that are installed systemwide,
-    run:
-
-        nbgrader extension uninstall
-
-    If you want to uninstall the extensions installed in your user
-    environment, use:
-
-        nbgrader extension uninstall --user
-
-    To uninstall only a specific extension, you can pass the name of the
-    extension you want to uninstall as an argument, e.g.:
-
-        nbgrader extension uninstall create_assignment
-        nbgrader extension uninstall assignment_list
-
-    """
-
-    destination = Unicode('')
-
-    def _classes_default(self):
-        return [ExtensionUninstallApp, UninstallNBExtensionApp]
-
-    def excepthook(self, etype, evalue, tb):
-        format_excepthook(etype, evalue, tb)
-
-    def start(self):
-        nbextensions_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'nbextensions'))
-        extra_args = self.extra_args[:]
-
-        # install the create_assignment extension
-        if len(extra_args) == 0 or "create_assignment" in extra_args:
-            self.log.info("Uninstalling create_assignment extension")
-            self.extra_args = ["create_assignment"]
-            self.uninstall_extensions()
-
-        # install the assignment_list extension
-        if sys.platform != 'win32' and (len(extra_args) == 0 or "assignment_list" in extra_args):
-            self.log.info("Uninstalling assignment_list extension")
-            self.extra_args = ["assignment_list"]
-            self.uninstall_extensions()
-
-
-class ExtensionActivateApp(EnableNBExtensionApp, NbGrader):
+_compat_message = """
+The installation of the nbgrader extensions are now managed through the
+`jupyter nbextension` and `jupyter serverextension` commands.
 
-    name = u'nbgrader-extension-activate'
-    description = u'Activate the nbgrader extension'
+To install and enable the nbextensions (assignment_list and create_assignment) run:
 
-    flags = {}
-    aliases = {}
+    $ jupyter nbextension install --sys-prefix --py nbgrader
```

Comment:
Ok, so just to be clear, all the extensions will always be *installed*, but we can give people different instructions for how to selectively *enable* extensions?

It would be great if you can update the documentation to be really explicit about that -- i.e., have one example for how to install and activate all the extensions, and then another for how to activate just one of the extensions.